packages/material-ui-lab/src/SpeedDialAction/SpeedDialAction.js

```diff
@@ -78,22 +85,55 @@ class SpeedDialAction extends React.Component {
       ...other
     } = this.props;
 
-    let clickProp = { onClick };
+    let clickProps = { onClick };
     if (typeof document !== 'undefined' && 'ontouchstart' in document.documentElement) {
```

Comment:
You're going with `const` in `SpeedDial` but `let` here. The same approach as in `SpeedDial` should be sufficient.